cfp.md

```diff
@@ -34,18 +34,57 @@ Contributions may overlap with any of the following areas:
 - Time-predictable hardware
 
 More background about this workshop and its motivation can be found [here]({% link about.md %}).
+
 ## Important Dates
-TBA
+- July 8, 2024: Paper submission deadline
+- Aug 5, 2024: Author notification for the first round of reviews
+- Aug 12, 2024: Camera-ready and video graphical abstract deadline
+- Aug 19, 2024: Final acceptance notification
+- TBD (One day during ESWEEK, the week of September 29 - October 4, 2024): Paper presentations
```

Comment:
wasn't this set to Thursday or Friday now?

Reply:
Thanks for pointing this out. I updated this with more specific tentative dates (Thu, Oct 3 or Fri, Oct 4), although it's not finalized yet.